datafusion/physical-expr/src/aggregate/first_last.rs

```diff
@@ -213,6 +235,7 @@ struct FirstValueAccumulator {
     ordering_req: LexOrdering,
     // Stores whether incoming data already satisfies the ordering requirement.
     requirement_satisfied: bool,
+    is_ignore_null: bool,
```

Comment:
It might be good to add a comment like above parameters.

Reply:
Added. Thanks